src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -3196,6 +3255,9 @@ def select(
         run_check_acceptability : bool
             Option to check acceptable range of the values of
             required parameters after  downselecting data on this object.
+        warn_spacing : bool
+            Option to raise warnings about spacing that would prevent writing to
+            calfits file-format. Default is True.
```

Comment:
calfits -> beamfits

Reply:
Fixed in e42deb6a26a1f3e58ccfe3436a40ca41022ccea0.